pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/ClinicalKitDao.java

```diff
@@ -29,8 +33,24 @@ public class ClinicalKitDao {
                     + "LEFT JOIN ddp_participant as p on (p.participant_id = inst.participant_id) "
                     + "LEFT JOIN ddp_instance as ddp on (ddp.ddp_instance_id = p.ddp_instance_id) "
                     + "LEFT JOIN sm_id_type sit on (sit.sm_id_type_id = sm.sm_id_type_id) "
-                    + "LEFT JOIN kit_type ktype on ( sit.kit_type_id = ktype.kit_type_id) WHERE sm.sm_id_value = ? "
-                    + "AND NOT sm.deleted <=> 1 ";
+                    + "LEFT JOIN kit_type ktype on ( sit.kit_type_id = ktype.kit_type_id) "
+                    + "WHERE sm.sm_id_value = ? AND NOT sm.deleted <=> 1 ";
+    private static final String SQL_GET_CLINICAL_TISSUE_BY_DDP_PARTICIPANT_ID = "SELECT kit.received_date FROM ddp_kit "
```

Comment:
How about naming the variable so it includes the fact that the query requires receipt?  Something like `SQL_GET_RECEIVED_TISSUE_BY_DDP_PARTICIPANT_ID`?  For this and L49,   I'm unsure of what it is about the query that makes it "clinical" vs. "sample has been received".

Reply:
normal samples wouldn't have a received date at all. we only know about clinical tissues being received per mercury call